src/test/rustdoc/normalize-assoc-item.rs

```diff
@@ -1,68 +0,0 @@
-// ignore-tidy-linelength
-// aux-build:normalize-assoc-item.rs
-// build-aux-docs
```

Comment:
Since we're going with low churn, can you mark this as `// ignore-test` instead of deleting it?

Reply:
done